queries.sql

```diff
@@ -156,4 +156,99 @@ SELECT owners.full_name , COUNT(*)
   INNER JOIN animals
   ON owners.id = animals.owner_id
   GROUP BY owners.full_name
-  ORDER BY count DESC LIMIT 1;
\ No newline at end of file
+  ORDER BY count DESC LIMIT 1;
+
+  -- Write queries to answer the following:
+-- [X] Who was the last animal seen by William Tatcher?
+SELECT vets.name AS "Vet Name", animals.name AS "Animal Name", visits.date_of_visit
+  FROM vets
+  INNER JOIN visits
+    ON vets.id = visits.vet_id
+  INNER JOIN animals
+    ON visits.animal_id = animals.id
+  WHERE vets.name = 'William Tatcher'
+  ORDER by date_of_visit DESC LIMIT 1;
+
+
+  -- [X] How many different animals did Stephanie Mendez see?
+SELECT vets.name AS "Vet Name", COUNT(DISTINCT animals.name)
+   FROM vets
+   INNER JOIN visits
+     ON vets.id = visits.vet_id
+   INNER JOIN animals
+     ON visits.animal_id = animals.id
+   WHERE vets.name = 'Stephanie Mendez'
+   GROUP BY vets.name;
+
+   -- [X] List all vets and their specialties, including vets with no specialties.
+SELECT vets.name AS "Vet Name", species.name AS "Species"
+  FROM vets
+  FULL OUTER JOIN specializations
+    ON vets.id = specializations.vet_id
+  FULL OUTER JOIN species
+    ON species.id = specializations.species_id;
+
+    -- [X] List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+SELECT animals.name AS "Animal", visits.date_of_visit
+  FROM animals
+  JOIN visits
+    ON animals.id = visits.animal_id
+  WHERE visits.vet_id = (SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez')
+  AND
+  visits.date_of_visit BETWEEN '2020-04-01' AND '2020-08-30';
+
+-- [X] What animal has the most visits to vets?
+SELECT animals.name, COUNT(*)
+  FROM animals
+  JOIN visits
+    ON animals.id = visits.animal_id
+  GROUP BY animals.name
+  ORDER BY count DESC LIMIT 1;
+
+  -- [X] Who was Maisy Smith's first visit?
+SELECT vets.name AS "Vet", animals.name AS "Animal", visits.date_of_visit
+  FROM vets
+  JOIN visits
+    ON vets.id = visits.vet_id
+  JOIN animals
+    ON visits.animal_id = animals.id
+  WHERE vets.name = 'Maisy Smith'
+  ORDER BY date_of_visit LIMIT 1;
```

Comment:
Great job @RichardChileya 🤝

- [ ] [Optional] You may kindly add a screenshot of Who was Maisy Smith's first visit?